packages/fxa-auth-server/lib/senders/emails/templates/verifyPrimary/index.txt

```diff
@@ -0,0 +1,13 @@
+verifyPrimary-title = "Verify primary email"
+
+verifyPrimary-description = "A request to perform an account change has been made from the following device:"
+<%- include('/partials/location/index.txt') %>
+
+verifyPrimary-action-plaintext = "Verify email:"
+<%- link %>
+
+verifyPrimary-post-verify = "Once verified, account changes like adding a secondary email will become possible from this device."
```

Comment:
This doesn't appear to be in the original plaintext version. I am okay with adding it in since it makes sense, but just want to acknowledge this.

Reply:
That also seems like a mistake to me on the old templates. I'll add it to that doc I mentioned in another comment.